firmware/src/sts_token.hpp

```diff
@@ -58,8 +58,8 @@ void STStoken_decode() {
   String identifier = sts_third.substring(6, 7);
   String sts_encode = sts_data.substring(15, 20);
   long tariff_MT_NO = sts_encode.toInt();
-  long tariff_gotten = ((tariff_MT_NO) - (meter_no * multiplier));
-  long trueMT_NO = tariff_MT_NO - tariff;
+  // long tariff_gotten = ((tariff_MT_NO) - (meter_no * multiplier));
```

Comment:
If you comment out a line, please add a comment saying why this is commented out.